frontend/src/component/feature/Dependencies/DependenciesUpgradeAlert.tsx

```diff
@@ -0,0 +1,22 @@
+import { Alert } from '@mui/material';
+
+export const DependenciesUpgradeAlert = () => {
+    return (
+        <Alert severity='warning'>
+            Remember to update your Unleash client! Feature dependencies require new
+            SDK versions. <DependenciesDocsLink />.
+        </Alert>
+    );
+};
+
+const DependenciesDocsLink = () => {
+    return (
+        <a
+            href='https://docs.getunleash.io/reference/dependent-features#client-sdk-support'
+            target='_blank'
+            rel='noreferrer'
+        >
+            Read more
```

Comment:
This link text doesn't say anything about what the link points to. It should be possible to understand what a link points to without any context. Can we do something like this instead?
```suggestion
            Client SDK support for feature dependencies
```

Potentially preface the link (outside of the `a` tag) with "Read more about"?

Reply:
fixing